assets/css/index.css

```diff
@@ -17,7 +16,7 @@ footer {
 }
 
 .sidebar-list a {
-  font-family: 'Merienda', cursive;
+  
```

Comment:
Instead of removing Merienda, replace it with "Syne" please.

Reply:
Done, please check :)